Gems/AtomLyIntegration/EditorModeFeedback/Code/Source/EditorModeFeedbackSystemComponent.h

```diff
@@ -40,8 +42,16 @@ namespace AZ
             // EditorModeFeedbackInterface overrides ...
             bool IsEnabled() const override;
 
-        private:
-            //! Settings registery override for enabling/disabling editor mode feedback.
+        private:            
+            ///////////////////////////////////////////////////////////////////////
```

Comment:
nit: Can we not add the //////////// lines, they just add noise and visual clutter in my mind, we don't have them in Editor code at all anymore for the most part